src/sensors/heading/gyroscope/GY50.cpp

```diff
@@ -64,24 +61,25 @@ void GY50::attach()
         return;
     }
 
+    static const uint8_t controlRegister1 = 0x20;
+    static const uint8_t controlRegister2 = 0x21;
+    static const uint8_t controlRegister3 = 0x22;
+    static const uint8_t controlRegister4 = 0x23;
+    static const uint8_t controlRegister5 = 0x24;
+
     mRuntime.i2cInit();
     // Enable z and turn off power down
-    static constexpr uint8_t kControlRegister1 = 0x20;
-    writeL3G4200DRegister(kControlRegister1, 0b00001100); // NOLINT(readability-magic-numbers)
+    writeL3G4200DRegister(controlRegister1, 0b00001100);
     // If you'd like to adjust/use the HPF, you can edit the line below to configure CTRL_REG2
-    static constexpr uint8_t kControlRegister2 = 0x21;
-    writeL3G4200DRegister(kControlRegister2, 0b00000000); // NOLINT(readability-magic-numbers)
+    writeL3G4200DRegister(controlRegister2, 0b00000000);
     // Configure CTRL_REG3 to generate data ready interrupt on INT2
     // No interrupts used on INT1, if you'd like to configure INT1
     // or INT2 otherwise, consult the datasheet
-    static constexpr uint8_t kControlRegister3 = 0x22;
-    writeL3G4200DRegister(kControlRegister3, 0b00001000); // NOLINT(readability-magic-numbers)
+    writeL3G4200DRegister(controlRegister3, 0b00001000);
     // CTRL_REG4 controls the full-scale range, among other things
-    static constexpr uint8_t kControlRegister4 = 0x23;
-    writeL3G4200DRegister(kControlRegister4, 0b00110000); // NOLINT(readability-magic-numbers)
+    writeL3G4200DRegister(controlRegister4, 0b00110000);
```

Comment:
:warning: **readability-magic-numbers** :warning:
0b00110000 is a magic number; consider replacing it with a named constant